zebra/irdp_main.c

```diff
@@ -234,7 +234,8 @@ int irdp_send_thread(struct thread *t_advert)
       }
 
   tmp = irdp->MaxAdvertInterval-irdp->MinAdvertInterval;
-  timer =  (random () % tmp ) + 1;
+  assert (tmp > 0);
+  timer = (random () % tmp) + 1;
   timer = irdp->MinAdvertInterval + timer;
```

Comment:
this was probably supposed to be `timer = random() % (tmp + 1)` in fact

(way to go, 13 years for a div-by-0 :D)